convert2rhel/utils.py

```diff
@@ -620,4 +627,52 @@ def set_locale():
     os.environ.update({"LC_ALL": "C", "LANG": "C"})
 
 
+def options_from_config_files(config_opts, cfg_path=None):
+    """Get the options from config file.
+
+    The order is:
+    1. if filename is set then load from it
+    2. ~/.convert2rhel.ini
+    3. /etc/convert2rhel.ini
+
+    Return config_opts filled with loaded values.
+    """
+    headers = ["subscription_manager"]  # supported sections in config file
+    config_file = configparser.ConfigParser()
+    paths = CONFIG_PATHS
+    # create dict with all supproted options, all of them set to None
+    # needed for avoiding problems with files priority
+    supproted_opts = {"password": None, "activation_key": None}
```

Comment:
Adding support for new option will be just adding the name there. And if needed the new header upper.